data_steward/cdr_cleaner/cleaning_rules/clean_ppi_numeric_fields_using_parameters.py

```diff
@@ -71,17 +59,19 @@
     observation_datetime,
     observation_type_concept_id,
 CASE
-    WHEN observation_concept_id IN (1585889, 1585890) AND (value_as_number < 0 OR value_as_number > 20) THEN NULL
+    WHEN observation_concept_id IN (1585889, 1585890) AND (value_as_number < 0) THEN NULL
```

Comment:
You still need to leave the value_as_number  > 20 here, so the values for the 1585890 will still be set to NULL.